osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/application/services/BundleService.java

```diff
@@ -69,6 +70,9 @@
     @Autowired
     private FirmwareService firmwareService;
 
+    @Autowired
+    private EventService eventService;
+
     public BundleService() {
         // Parameterless constructor required for transactions...
```

Comment:
in this Bean, @Transactional is set at the class level. Maybe a good idea would be to check whether the 2 public methods can get their own annotation. This will make the bean eligible for constructor injection and improved testing capabilities.

Reply:
I added the Transactional to the methods and made a constructor that is used for constructor injection

Reply:
we shouldn't introduce major sonar issues